lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -799,3 +799,113 @@ pcmk__block_colocated_starts(pe_action_t *action, pe_working_set_t *data_set)
         }
     }
 }
+
+/*!
+ * \internal
+ * \brief Determine what colocation constraint should affect at this point
```

Comment:
Should this say `Determine what colocation constraint should be in effect at this point`?  Or something else?  It seems to be missing a word as written.

Reply:
Oh I see, constraint is a parameter.  Can you make it `\p constraint` or `the constraint parameter` or something?

Reply:
done